meilisearch-http/src/routes/setting.rs

```diff
@@ -63,11 +59,7 @@ pub async fn get_all(ctx: Request<Data>) -> SResult<Response> {
             .iter()
             .map(|s| (*s).to_string())
             .collect::<HashSet<String>>();
-        if attrs.is_empty() {
-            None
-        } else {
-            Some(attrs)
-        }
+        Some(attrs)
```

Comment:
I would prefer you remove this useless `Some` wrapping and put it at Settings creation, like the others.